crypto/multikey/kind_test.go

```diff
@@ -193,7 +288,7 @@ func TestRemoveDuplicates(t *testing.T) {
 	for _, tt := range tests {
 		t.Run("remove-duplicates", func(t *testing.T) {
 			got := removeDuplicates(tt.in)
-			if !assert.Equal(tt.want, got) {
+			if !assertion.Equal(tt.want, got) {
```

Comment:
If you change to if !assert.Equal(t, tt.want, got) { this will give more information about what test case failed.